tests/test_object_detection.py

```diff
@@ -96,6 +101,9 @@ def test_inference_autocast(self):
             self.assertLess(np.abs(boxes[0].to(torch.float32).cpu().detach().numpy() - expected_location).max(), 5)
 
     def test_inference_hpu_graphs(self):
+        if is_eager_mode():
```

Comment:
could you use pytest skipif decorator instead: https://docs.pytest.org/en/7.1.x/how-to/skipping.html#id1

Reply:
pushed another commit to implement this.